phase-0-tracks/ruby/iteration.rb

```diff
@@ -0,0 +1,101 @@
+5.3
+
+actors = ["H. Ford", "S.C", "M.K", "S.W"]
+roles = {"H.F" => "ij", "S.C" => "jb", "M.K" => "bm", "S.W" => "er"}
+
+
+ actors.each do |x|
+ puts x
+end
+
+ actors.map! do |x|
+  x.split(//)
+end
+
+p actors
+
+new = roles.each do |x, y|

```

Comment:
Consider what setting `roles.each` is returning.  `.each` runs the block and returns the enumerator.  This means that it runs whatever code is inside of the block that you have written and then returns the data structure that you are calling the enumerable `.each`.

In short it is just returning the data type you are iterating on.

```
new == roles
```